### PR TITLE
Add new GCArticles pages into WAF admin rules

### PR DESCRIPTION
**Pull request by jimleroyer**
# Summary | Résumé

Add new GCArticles pages into WAF admin rules:

* /autres-services
* /confidentialite-old
* /other-services
* /privacy-old
* /security-old
* /securite-old

# Test instructions | Instructions pour tester la modification

Waffles check should pass!

### CODE REVIEW

File: aws/eks/admin_waf_regex_patterns.tf [hunk 1]
```diff
@@ -33,14 +33,20 @@ resource "aws_wafv2_regex_pattern_set" "re_admin" {
...
   }
 
   # GCA routes
   regular_expression {
     regex_string = "/terms|/conditions-dutilisation|/personalisation-guide|/guide-personnalisation|/message-delivery-status|/etat-livraison-messages|/formatting-guide|/guide-mise-en-forme|/spreadsheets|/feuille-de-calcu"
   }
 
+
+  # GCA routes
+  regular_expression {
```
Comment (andrewleith) [inline aws/eks/admin_waf_regex_patterns.tf:46]: Just curious, why are there so many separate `regular_expression` objects.  Is this due to a limitation or is there an advantage of some sort of breaking it out?

File: aws/eks/admin_waf_regex_patterns.tf [hunk 2]
```diff
@@ -33,14 +33,20 @@ resource "aws_wafv2_regex_pattern_set" "re_admin" {
 
   # GCA routes
   regular_expression {
-    regex_string = "/home|/accueil|/why-gc-notify|/pourquoi-gc-notification|/features|/fonctionnalites|/guidance|/guides-reference|/security|/securite|/privacy|/confidentialite|/accessibility|/accessibilite"
+    regex_string = "/home|/accueil|/why-gc-notify|/pourquoi-gc-notification|/features|/fonctionnalites|/guidance|/guides-reference|/securit.+|/privacy.*|/confidentialite.*|/accessibility|/accessibilite"
```
[the remark this answers is not in the document]
Comment (jimleroyer) [reply aws/eks/admin_waf_regex_patterns.tf:36]: For all the mentioned paths you mean?

File: aws/eks/admin_waf_regex_patterns.tf (hunk 1 above)
Comment (jimleroyer) [reply aws/eks/admin_waf_regex_patterns.tf:46]: I followed what was present in the file and can only assume. For readability, it's certainly easier this way.

File: aws/eks/admin_waf_regex_patterns.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrewleith) [reply aws/eks/admin_waf_regex_patterns.tf:36]: Sorry I meant, should we even have them at all?  Won't it allow extra paths through?

**Review by andrewleith — APPROVED** [review]

**Pull request merged**